### ISSUE TITLE
Private files image styles

### ISSUE THREAD

**Issue by webdobe** [wodby/drupal-php#32]
I am unable to get private files to generate image styles.  Public ones work fine.  Uploading private files work fine.  Generating image styles does not.

**Comment by csandanov** [comment]
Try adding the following location and see if it helps.
```
location ~* /system/files/styles/ {
    access_log {{ getenv "NGINX_STATIC_CONTENT_ACCESS_LOG" "off" }};
    expires {{ getenv "NGINX_STATIC_CONTENT_EXPIRES" "30d" }};
    try_files $uri @drupal;
 }
```

**Comment by csandanov** [comment]
Try the latest drupal-nginx image (non-tagged) with this fix https://github.com/wodby/drupal-nginx/issues/25

**Comment by webdobe** [comment]
Ok it was wodby/drupal-nginx#25 what I was missing.  I had added the other based on other drupal threads. was missing the included params in the system files section.

So both of these together are what solved my issues.

**Comment by csandanov** [comment]
@webdobe to be clear, what did you do in addition to this include fix in drupal-nginx?

**Comment by csandanov** [comment]
@webdobe ping

**Comment by webdobe** [comment]
It appears that the things that fixed it were fixes found here: wodby/drupal-nginx#25

I had already applied before your comment: 
location ~* /system/files/styles/ {
    access_log {{ getenv "NGINX_STATIC_CONTENT_ACCESS_LOG" "off" }};
    expires {{ getenv "NGINX_STATIC_CONTENT_EXPIRES" "30d" }};
    try_files $uri @drupal;
 } 

I didn't test not having that.

**Comment by csandanov** [comment]
Ok, so the [`include fastcgi_params;`](https://github.com/wodby/drupal-nginx/commit/e34b44e51dbefe292370540f6a298a835ca63522) fix did the trick.
